packaging/aznfs/DEBIAN/prerm

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Exit on error.
+set -e
+
```

Comment:
we should check for mountmap file and if it has existing mounts, warn the user and ask for confirmation that there are Azure Blob NFS mounts mounted using the aznfs mount helper, they will not be tracked, are you sure you want to continue? stuff like that